src/python/pants/notes/2.16.x.md

```diff
@@ -144,6 +144,55 @@ The deprecated `Platform.current` has been removed. Instead, have the affected r
 
 The `ToolCustomLockfile` and `ToolDefaultLockfile` classes [have been removed](https://github.com/pantsbuild/pants/pull/17843).
 
+## 2.16.0a1 (Mar 24, 2023)
+
+### New Features
+
+* Always provide Python-for-Pants-scripts (Cherry-pick of #18433) ([#18495](https://github.com/pantsbuild/pants/pull/18495))
+
+* Expose environment information on process metadata (Cherry-pick of #18458) ([#18463](https://github.com/pantsbuild/pants/pull/18463))
+
+### User API Changes
+
+* Upgrade Pex to 2.1.130. (Cherry-pick of #18576) ([#18578](https://github.com/pantsbuild/pants/pull/18578))
+
+* Deprecate old-style custom tool lockfiles. (#18494) ([#18498](https://github.com/pantsbuild/pants/pull/18498))
+
+* Upgrade Pex to 2.1.129. (Cherry-pick of #18472) ([#18473](https://github.com/pantsbuild/pants/pull/18473))
```

Comment:
Killing the 2.1.129 and 2.1.128 entries would be good, but not a big deal either.